src/domains/citizen-auth-app/99_locals.tf

```diff
@@ -31,3 +29,12 @@ locals {
   storage_account_notifications_queue_userslogin         = "userslogin"
   storage_account_notifications_queue_push_notifications = "push-notifications"
 }
+
+# Region ITN
+locals {
+  itn_location       = "italynorth"
+  itn_location_short = "itn"
+  project_itn        = "${var.prefix}-${var.env_short}-${var.domain}-${local.itn_location_short}-${var.instance}"
+  product_itn        = "${var.prefix}-${var.env_short}"
```

Comment:
this is duplicated of `product`

Reply:
Done in [0bfca36](https://github.com/pagopa/io-infra/pull/1037/commits/0bfca3668b70b49e4886723550333453ff842e9d)